data/lib/src/datasource_impl/upload_request_entry_datasource_impl.dart

```diff
@@ -59,4 +70,43 @@ class UploadRequestEntryDataSourceImpl implements UploadRequestEntryDataSource {
       });
     });
   }
+
+  @override
+  Future<List<DownloadTaskId>> downloadUploadRequestEntries(List<UploadRequestEntry> entries, Token token, Uri baseUrl) async {
+    var externalStorageDirPath;
+    if (Platform.isAndroid) {
+      externalStorageDirPath = await ExtStorage.getExternalStoragePublicDirectory(ExtStorage.DIRECTORY_DOWNLOADS);
+    } else if (Platform.isIOS) {
+      externalStorageDirPath = (await getApplicationDocumentsDirectory()).absolute.path;
+    } else {
+      throw DeviceNotSupportedException();
+    }
+
+    final taskIds = await Future.wait(
+        entries.map((entry) async => await FlutterDownloader.enqueue(
+        url: Endpoint.uploadRequestsEntriesRoute
+            .downloadServicePath(entry.uploadRequestEntryId.uuid)
+            .generateDownloadUrl(baseUrl),
+        savedDir: externalStorageDirPath,
+        fileName: entry.name,
+        headers: {Constant.authorization: 'Bearer ${token.token}'},
+        showNotification: true,
+        openFileFromNotification: true)));
+
+    return taskIds
+        .where((id) => id != null)
+        .map((taskId) => DownloadTaskId(taskId!))
+        .toList();  }
```

Comment:
weird indentation there

Reply:
you mean indentation should be 2 instead of 4 space? @imGok

Reply:
I mean the last bracket should be in new line